applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_plane_stress_uncoupled_shear.cpp

```diff
@@ -62,7 +62,23 @@ int ElasticIsotropicPlaneStressUncoupledShear::Check(
     const ProcessInfo& rCurrentProcessInfo
 )
 {
-    LinearPlaneStress::Check(rMaterialProperties, rElementGeometry, rCurrentProcessInfo);
+    if (YOUNG_MODULUS.Key() == 0 || rMaterialProperties[YOUNG_MODULUS] < 0.0)
+    {
+        KRATOS_ERROR << "YOUNG_MODULUS has Key zero or invalid value " << std::endl;
```

Comment:
Use the checks in includes/checks.h

~~~cpp
KRATOS_CHECK_VARIABLE_KEY(YOUNG_MODULUS)
~~~
and KRATOS_ERROR_IF